ui/components/multichain/pages/send/components/quote-card/index.tsx

```diff
@@ -0,0 +1,236 @@
+            </Text>
+          </Box>
+          <Box display={Display.Flex} alignItems={AlignItems.center}>
+            <Text
+              display={Display.Flex}
+              color={TextColor.textAlternative}
+              marginRight={'auto'}
+              gap={1}
+              alignItems={AlignItems.center}
+              variant={TextVariant.bodySm}
+            >
+              {t('transactionDetailGasHeading')}
+              <Tooltip
+                interactive
+                position="left"
+                containerClassName="info-tooltip__tooltip-container"
+                tooltipInnerClassName="info-tooltip__tooltip-content"
+                tooltipArrowClassName="info-tooltip__left-tooltip-arrow"
+                style={{ display: 'flex', height: '12px', aspectRatio: '1' }}
+                html={
+                  <>
+                    <p>{t('swapGasFeesSummary', [translatedNetworkName])}</p>
+                    <p>{t('swapGasFeesDetails')}</p>
+                    <p>
+                      <a
+                        onClick={() => {
+                          /* istanbul ignore next */
+                          trackEvent({
+                            event: 'Clicked "Gas Fees: Learn More" Link',
+                            // TODO: update for swap and send
+                            category: MetaMetricsEventCategory.Swaps,
+                          });
+                          global.platform.openTab({
+                            url: GAS_FEES_LEARN_MORE_URL,
+                          });
+                        }}
+                        target="_blank"
+                        rel="noreferrer"
+                      >
+                        {t('swapGasFeesLearnMore')}
+                      </a>
+                    </p>
+                  </>
+                }
+                theme="tippy-tooltip-info"
+              >
+                <InfoTooltipIcon fillColor="var(--color-icon-alternative)" />
+              </Tooltip>
+            </Text>
+            <Box display={Display.Flex} marginLeft={'auto'}>
+              <Text variant={TextVariant.bodySm}>{formattedEthGasFee}</Text>
+              {formattedFiatGasFee && (
+                <Text
+                  color={TextColor.textAlternative}
+                  variant={TextVariant.bodySm}
+                  marginLeft={1}
+                >
+                  ≈ {formattedFiatGasFee}
+                </Text>
+              )}
+            </Box>
+          </Box>
+        </Box>
+      )}
+      {/* FEE INFO */}
+      {bestQuote && (
+        <Text color={TextColor.textAlternative} variant={TextVariant.bodySm}>
+          {t('swapIncludesMMFeeAlt', [bestQuote?.fee])}
+        </Text>
+      )}
+      {/* TOS LINK; doubles as anchor for scroll ref */}
+      <ButtonLink
+        variant={TextVariant.bodySm}
+        href={CONSENSYS_TERMS_OF_USE}
+        target="_blank"
+        className="quote-card__TOS"
+        disableUnderline
```

Comment:
If the underline on hover must be overridden can we do it with a custom className instead of updating the component api itself?